docs/reference/module-options.md

```diff
@@ -0,0 +1,148 @@
+# Module configuration options
+
+Most Apostrophe modules, both core and project-level, can be configured with settings that influence functionality without having to write custom Javascript. The sections below describe the options available in all modules as well as those specific to certain module types.
+
+All the settings described here are placed in a module's `options` configuration object. This often added in the respective `index.js` file, but can also be added where the module is instantiated in the `app.js` file.
+
+`index.js` example:
+```javascript
+// In `modules/article/index.js`
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  // 👇 Module configuration options
+  options: {
+    alias: 'article',
+    label: 'Article',
+    pluralLabel: 'Articles',
+    quickCreate: true
+  },
+  // Other settings, such as `fields`
+}
+```
+
+`app.js` example:
+```javascript
+// In `app.js`
+require('apostrophe')({
+  shortName: 'bowling-league-site', // Unique to your project
+  modules: {
+    article: {
+      extend: '@apostrophecms/piece-type',
+      // 👇 Module configuration options
+      options: {
+        alias: 'article',
+        label: 'Article',
+        pluralLabel: 'Articles',
+        quickCreate: true
+      },
+      // Other settings, such as `fields`
+    }
+  }
+});
+```
+
+## Options for any module
+
+- [`alias`](#alias)
+- [`browser`](#browser)
+<!-- - [`components`](#components) -->
+- [`name`](#name)
+- [`templateData`](#templatedata)
+
+### `alias`
+
+Set to a string, the `alias` value will be applied to the `apos` object (accessible in many places) as a quick reference to the module.
```

Comment:
Should we be linking to `apos` object? Can we?

Reply:
We don't have that yet...

Reply:
"With the exception of modules present in the apostrophe core at the time of the 3.x stable release, modules distributed via npm should never set alias for themselves or assume that it has been set."

Reply:
Added this in a note below.